identity/credentials.go

```diff
@@ -120,7 +120,7 @@ func (c CredentialsType) ToUiNodeGroup() node.UiNodeGroup {
 var AllCredentialTypes = []CredentialsType{
 	CredentialsTypePassword,
 	CredentialsTypeOIDC,
-	CredentialsTypeSAML,
+	// CredentialsTypeSAML, placeholder for the OEL version
```

Comment:
Do we have a way do re-inject this credential type in OEL?

Reply:
The SAML type was never part of this slice. I just added it because I was confused at first why it is not part, but it really should not be.